### ISSUE TITLE
Error on latest beta with controller type MQTT and UFOR11

### ISSUE THREAD

**Issue by gurglingtonic** [litinoveweedle/SmartIR#98]
Tested the latest beta, this error appear at "Check Configuration":-

> Invalid config for 'smartir' from integration 'fan' at packages/configuration/fan.yaml, line 37: required key 'remote_entity' not provided
> Invalid config for 'smartir' from integration 'fan' at packages/configuration/fan.yaml, line 38: Values are not equal: value:UFOR11 != target:Broadlink for dictionary value 'controller_data->controller_type', got 'UFOR11'
> Invalid config for 'smartir' from integration 'fan' at packages/configuration/fan.yaml, line 39: 'mqtt_topic' is an invalid option for 'fan.smartir', check: controller_data->mqtt_topic

> Invalid config for 'smartir' from integration 'fan' at packages/configuration/fan.yaml, line 37: required key 'remote_entity' not provided
> Invalid config for 'smartir' from integration 'fan' at packages/configuration/fan.yaml, line 38: Values are not equal: value:MQTT != target:Broadlink for dictionary value 'controller_data->controller_type', got 'MQTT'
> Invalid config for 'smartir' from integration 'fan' at packages/configuration/fan.yaml, line 39: 'mqtt_topic' is an invalid option for 'fan.smartir', check: controller_data->mqtt_topic

My fan.yaml config:-
```
  - platform: smartir
    name: Ceiling Fan - Bedroom II
    unique_id: ceiling_fan_bedroom_2
    device_code: 16
    controller_data:
      controller_type: UFOR11
      mqtt_topic: zigbee2mqtt_lan/Bedroom 2 - IR Remote/set
    power_sensor: switch.bedroom_2_lights_fan_switch_left
    power_sensor_restore_state: true
```

```
  - platform: smartir
    name: Ceiling Fan - Bedroom II
    unique_id: ceiling_fan_bedroom_2
    device_code: 16
    controller_data:
      controller_type: MQTT
      mqtt_topic: zigbee2mqtt_lan/Bedroom 2 - IR Remote/set/ir_code_to_send
    power_sensor: switch.bedroom_2_lights_fan_switch_left
    power_sensor_restore_state: true
```

May need to update the checker for MQTT and UFOR11 or please advise otherwise.

**Comment by Barneybaer84** [comment]
Same here.

**Comment by mirkokrajco** [comment]
I have some problem.... 

my config:

climate:
  - platform: smartir
    name: Obyvačka klima
    unique_id: obyvacka_klima
    device_code: 1281
    controller_data:
      controller_type: UFOR11
      mqtt_topic: zigbee2mqtt/Dialkove ovladanie 1/set
    temperature_sensor: sensor.teplota_obyvacka_temperature
    humidity_sensor: sensor.teplota_obyvacka_humidity

  - platform: smartir
    name: Spálňa klima
    unique_id: spalna_klima
    device_code: 1281
    controller_data:
      controller_type: UFOR11
      mqtt_topic: zigbee2mqtt/Dialkove ovladanie 2/set


error:

**Invalid config for 'climate' from integration 'smartir' at configuration.yaml, line 383: required key 'remote_entity' not provided, please check the docs at https://github.com/litinoveweedle/SmartIR Invalid config for 'climate' from integration 'smartir' at configuration.yaml, line 384: Values are not equal: value:UFOR11 != target:Broadlink for dictionary value 'controller_data->controller_type', got 'UFOR11', please check the docs at https://github.com/litinoveweedle/SmartIR Invalid config for 'climate' from integration 'smartir' at configuration.yaml, line 385: 'mqtt_topic' is an invalid option for 'smartir.climate', check: controller_data->mqtt_topic, please check the docs at https://github.com/litinoveweedle/SmartIR
Invalid config for 'climate' from integration 'smartir' at configuration.yaml, line 393: required key 'remote_entity' not provided, please check the docs at https://github.com/litinoveweedle/SmartIR Invalid config for 'climate' from integration 'smartir' at configuration.yaml, line 394: Values are not equal: value:UFOR11 != target:Broadlink for dictionary value 'controller_data->controller_type', got 'UFOR11', please check the docs at https://github.com/litinoveweedle/SmartIR Invalid config for 'climate' from integration 'smartir' at configuration.yaml, line 395: 'mqtt_topic' is an invalid option for 'smartir.climate', check: controller_data->mqtt_topic, please check the docs at https://github.com/litinoveweedle/SmartIR**

**Comment by litinoveweedle** [comment]
- OMG, me too! What we shall do about it? :-) Litinoveweedle, fix it! 
- Yes Sir! I propose to fix it by repairing copy paste error in the controller_const.py on the line 37!
- Litinoveweedle you have permission to do it!
- Already done sir, in the #100 
- Great job Litinoveweedle, how the poor users can use it?
- The can copy the affected file from the master, until new beta is released. May I have proposal sir?
- Sure you can.
- I would appreciated if next time user instead of writing 'me to'  checked the code and proposed PR, as this one was really obvious to find out.
- We can only hope for that, Litinoveweedle.

**Comment by litinoveweedle** [comment]
Released in 1.18.0b3
